src/d2_docker/utils.py

```diff
@@ -296,13 +330,18 @@ def run_docker_compose(
         # Add ROOT_PATH from environment (required when run inside a docker)
         ("ROOT_PATH", ROOT_PATH),
         ("PSQL_ENABLE_QUERY_LOGS", "") if not enable_postgres_queries_logging else None,
+        ("GLOWROOT_PORT", "{}:4000".format(glowroot_port) if glowroot_port else "4000:4000") if (glowroot or glowroot_zip) else ("GLOWROOT_PORT", None),
```

Comment:
let's move this 4000 to some variable at the top of the file, to make it more visible.